web_tree_dynamic_colored_field/static/src/js/web_tree_dynamic_colored_field.js

```diff
@@ -10,21 +10,41 @@ odoo.define('web_tree_dynamic_colored_field', function (require) {
          *
          * @override
          */
-        _renderBody: function () {
+        _renderView: function () {
             if (this.arch.attrs.colors) {
-                var colorAttr = this.arch.attrs.colors.split(';')
-                    .filter(color => color.trim().startsWith('color_field'));
+                var colors_attr = this.arch.attrs.colors.split(';')
```

Comment:
It's not mandatory for the language.... but in OCA we still using semicolon.
```suggestion
                var colors_attr = this.arch.attrs.colors.split(';');
```